tools/get_coverage_ros2_pkg.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import requests
+import subprocess
+import sys
+import tempfile
+
+parser = argparse.ArgumentParser()
+parser.add_argument("jenkins_coverage_build", help="URL of a ci.ro2s.org build using coverage (i.e https://ci.ros2.org/job/ci_linux_coverage/182)")
```

Comment:
Nitpick: following the ROS 2 developer guide use single quotes except if the string literal contains them.

Same below.

Reply:
f2f652f

Reply:
Still some double quotes.

Reply:
I changed the only place I found where I'm not using single quotes inside the string, e5bddaa . Am I missing other occurrences? 4d4d9fe

Reply:
Only one more: https://github.com/ros2/ci/pull/476/files#diff-b160c856c85886178880e50860b3281fR76